File: src/data/snippets/for-loops/snippet.java
public static void main(String[] args) {
    // Print numbers from 1 to 20 with even or odd indication
    for (int num = 1; num <= 20; num++) {
        if (num % 2 == 0) {
            System.out.println(num + " is even");
        } else {
            System.out.println(num + " is odd");
        }
    }
}

File: src/data/snippets/for-loops/snippet.cs
// Print numbers from 1 to 20 with even or odd indication
for (int num = 1; num <= 20; num++)
{
    if (num % 2 == 0)
    {
        Console.WriteLine($"{num} is even");
    }
    else
    {
        Console.WriteLine($"{num} is odd");
    }
}
